jobs/wazuh-manager/monit

```diff
@@ -1,55 +1,97 @@
-check process ossec-analysisd
-  matching "ossec-analysisd"
-  start program "/var/vcap/jobs/wazuh-manager/bin/wazuh_server_ctl ossec-analysisd start"
-  stop program "/var/vcap/jobs/wazuh-manager/bin/wazuh_server_ctl ossec-analysisd stop"
+check process wazuh-analysisd
+  matching "wazuh-analysisd"
+  start program "/var/vcap/jobs/wazuh-manager/bin/wazuh_server_ctl start"
+  stop program "/var/vcap/jobs/wazuh-manager/bin/wazuh_server_ctl stop"
```

Comment:
This is correct? bin / wazuh_server_ctl start '?

how it comes from this: bin / wazuh_server_ctl ossec-analysisd start '

leaves me wondering if the ... ossec-analysisd ... should not be preserved?

Reply:
The wazuh_server_ctl has also changed so the daemon name is not longer needed.

Reply:
Cool, LGTM, thks